storage/google/cloud/storage/blob.py

```diff
@@ -348,28 +370,23 @@ def download_to_file(self, file_obj, client=None):
         :raises: :class:`google.cloud.exceptions.NotFound`
         """
         client = self._require_client(client)
-        if self.media_link is None:  # not yet loaded
-            self.reload()
-
-        download_url = self.media_link
-
-        # Use apitools 'Download' facility.
-        download = Download.from_stream(file_obj)
-
-        if self.chunk_size is not None:
-            download.chunksize = self.chunk_size
-
+        # Get the download URL.
+        download_url = self._get_download_url()
```

Comment:
The old code called `self.reload()` to get the back-end-supplied properties populated.  This version doesn't do that, AFAICT.